core-bundle/src/Resources/contao/modules/Module.php

```diff
@@ -236,10 +236,10 @@ public function generate()
 		}
 
 		// Tag the module (see #2137)
-		if (System::getContainer()->has('fos_http_cache.http.symfony_response_tagger'))
+		if (!empty($tags = $this->getResponseCacheTags()) && System::getContainer()->has('fos_http_cache.http.symfony_response_tagger'))
```

Comment:
Should be the other way around :) We don't need to collect the tags if there's no response tagger.

Reply:
see 708a32aed64d7eaa2834c037ba88adc9c4579138